swupd/create_manifests.go

```diff
@@ -257,14 +257,51 @@ func processBundles(ui UpdateInfo, c config) ([]*Manifest, error) {
 
 func addUnchangedManifests(appendTo *Manifest, appendFrom *Manifest) {
 	for _, f := range appendFrom.Files {
-		if f.findFileNameInSlice(appendTo.Files) == nil {
-			if f.Name == IndexBundle {
-				// this is generated new each time
-				continue
-			}
-			appendTo.Files = append(appendTo.Files, f)
+		if f.Name == IndexBundle {
+			// this is generated new each time
+			continue
+		}
+		if f.Type == TypeManifest &&
+			f.findFileNameInSlice(appendTo.Files) != nil {
+			continue
+		}
+		if f.Type == TypeIManifest &&
+			f.findIManifestInSlice(appendTo.Files) != nil {
+			continue
+		}
+		appendTo.Files = append(appendTo.Files, f)
+	}
+}
+
+// writeIterativeManifests writes Iterative Manifests for all bundles in
+// newManifests and add them to the MoM.
+// Iterative Manifests are manifests with all file changes added in versions
+// greater than fromVersion.
+func (MoM *Manifest) writeIterativeManifests(newManifests []*Manifest, out string, fromVersion uint32) error {
+	var err error
+	// write manifests then add them to the MoM
+	for _, bMan := range newManifests {
+		//Don't write I manifest for full and new bundles
```

Comment:
Please make your comment spacing consistent. Leave a space after the `//` like you do everywhere else.

Reply:
ok. thanks